src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -121,18 +125,26 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
     /* will store a single page of block results */
     def processBlocksPage(results: Future[tezosNodeOperator.BlockFetchingResults]): Future[Int] =
       results.flatMap {
-        blocksWithAccounts =>
+        blocksWithAccounts => {
+
+          val originationAlter = parametersLense.modify(_.flatMap(toMichelsonScript[MichelsonExpression](_)))
```

Comment:
Could we push the Michelson parsing one level 'down', i.e. instead of Lorre, can we do it in `TezosNodeOperator` or somewhere else suitable? We should keep the entry point file as simple as possible.